charts/stable/local-ai/values.yaml

```diff
@@ -68,22 +66,24 @@ workload:
               path: /readyz
             startup:
               enabled: true
-              type: http
-              path: /readyz
+              type: tcp
           imageSelector: image
           env:
             ADDRESS: ":{{ .Values.service.main.ports.main.port }}"
             MODELS_PATH: "{{ .Values.persistence.models.mountPath }}"
             IMAGE_PATH: "{{ .Values.persistence.images.mountPath }}"
             BUILD_TYPE: "{{ .Values.localai.build_type }}"
-            REBUILD: "{{ .Values.localai.rebuild }}"
-            GO_TAGS: "{{ .Values.localai.go_tags }}"
-            CONTEXT_SIZE: "{{ .Values.localai.context_size }}"
+            # breaks chart if true, keep it false.
+            REBUILD: false
             DEBUG: "{{ .Values.localai.debug }}"
             CORS: "{{ .Values.localai.cors }}"
+            GALLERIES: "{{ toJson .Values.localai.galleries }}"
+            PRELOAD_MODELS: "{{ toJson .Values.localai.preload_models }}"
             CORS_ALLOW_ORIGINS: "{{ .Values.localai.cors_allow_origins }}"
 
 persistence:
+  varlog:
+    enabled: false
```

Comment:
```suggestion
```

Reply:
Can you also add on all ci/*.yaml files (Only there)

tmp:
  type: pvc
devshm:
  type: pvc
  
 Looks like CI is failing because of a small quota on emptyDirs